### ISSUE TITLE
Article Title and Sidebar Missing

### ISSUE THREAD

**Issue by Neverdusk** [worldanvil/worldanvil-bug-tracker#97]
**World Anvil Username**: Neverdusk

**World / Campaign / RPG Character / Manuscript Title**: Ocarina

**Feature**: Article Title and Sidebar

**Describe the Issue**
One of my articles is suddenly missing a title and sidebar. I recently became a Guild Member and noticed that one of my articles, which previously had a displayed title and sidebar, was now missing it on its page. I don't know how it happened, since I haven't edited the article's Sections, Design, or Navigation information. 

It now looks like a generic article, however, it has sidebar content filled in. The specific article is a Geography (specifically a Universe), with a title, type, description, alternate name, and several child articles. However only the "description" is visible. According to the preview (screenshot attached) their should at least be a title if not a sidebar.

This happened once previously to a newly-created article, which I simply remade. However, this current article has a lot of links connected to it that would be a lot of work to redo for a new article. I've created several articles since, including other Geography articles, and none of them have this error.

**To Reproduce**
1. Create article
2. Leave page
3. Return to page

**Expected behavior**
1. Create article
2. Leave page
3. Return to article
4. Article is properly displayed, with a title, description, and sidebar.

**Screenshots**
Editor Preview
![Capture2](https://user-images.githubusercontent.com/79533944/112410234-42c2c600-8cf1-11eb-97d7-af76a62b810f.png)

Article Page
![Capture2](https://user-images.githubusercontent.com/79533944/112409808-88cb5a00-8cf0-11eb-9d4e-b28b0b6ea8b4.png)

**Comment by iamromeo** [comment]
Hello @Neverdusk I am really sorry but with the data you have provided above I cannot reproduce what you see. 

Normally loss of a sidebar would mean there is an issue with the tags added to the article but I cannot confirm that due to the fact that I do not have the actual text. 

In terms of the header that is indeed odd, but there IS an option to remove the header, which once again, without knowing more about the way the article was setup I cannot confirm 

Please send our way more information so we can check this out.

**Comment by Kordishal** [comment]
The sidebar is most likely hidden because the sidebar hiding toggle is checked. Please check the articles sidebar settings (under Sections > Sidebar Sections way at the bottom.

**Comment by Neverdusk** [comment]
@iamromeo I'm sorry about the limited information. The article was made a few days ago, before I became a Guild Member. This error happened sometime between then and now, but I don't have an exact sequence of events that could have led up to it. If you need, I'll make the world public and link the article: https://www.worldanvil.com/w/ocarina-neverdusk/a/the-world-location

@Kordishal I checked the Sidebar options it was unchecked, as you said. I have no idea when that happened, because I don't believe I've ever edited that page. However, checking the option did restore the sidebar, so thank you for the fix! 

However, the article title's is still missing:
![Capture2](https://user-images.githubusercontent.com/79533944/112502540-8b659800-8d60-11eb-9f32-77fa05773e7a.png)

**Comment by Neverdusk** [comment]
After going through different settings, I found one called "Display the title of the article in presentation mode" (under Preferences) and checked it, and that seemed to solve the missing title. The weird part is, I've never looked at this page before, or edited any part of it - which is why I was so lost by the information disappearing. 

Perhaps the options somehow changing was a separate bug. I have noticed strange things happening, like random image artifacts appearing after saving a page (I think the editor actually saves the loading GIF that sometimes appears into the article), so this could be another one of those strange occurrences, but for now I've fixed the issue. Sorry for the inconvenience, and thank you very much for helping.
